src/components/Navbar/Navbar.vue

```diff
@@ -12,13 +12,23 @@ function handleDropdownButton(dropdown: boolean | string) {
 		openDropdown.value = dropdown;
 	}
 }
+
+const scrollY = ref<number>(0);
+
+onMounted(() => {
+	// eslint-disable-next-line no-undef -- we're in on mounted so it should be fine
```

Comment:
My bad, I didn't configure the globals

You can use the example from https://eslint.vuejs.org/user-guide/#configuration-eslint-config-js, set the globals as they show. You don't need the rules section or source type though